marklogic-data-hub-central/ui/src/components/entities/matching/matching-card.tsx

```diff
@@ -98,7 +98,7 @@ const MatchingCard: React.FC<Props> = (props) => {
         </Modal>;
 
     return (
-        <div className={styles.loaddataContainer}>
+        <div className={styles.loadContainer}>
```

Comment:
Should this still be loadContainer, even though it's for the match card? Perhaps a more general name - these are all steps, so "stepContainer"?

Reply:
Thanks, @rjrudin.